inc/class-wpseo-shortlinker.php

```diff
@@ -21,6 +21,17 @@ public static function get_encoded_query() {
 		return build_query( $shortlinker->collect_additional_shortlink_data() );
 	}
 
+	/**
+	 * Gets the shortlink's query params.
+	 *
+	 * @return array The shortlink's query params.
+	 */
+	public static function get_query_params() {
+		$shortlinker = new WPSEO_Shortlinker();
```

Comment:
Can you change this to `new self`, because you are referencing within the same class.